x/ibc-hooks/contracts/crosschain-swaps/README.md

```diff
@@ -35,9 +34,9 @@ of an IBC transfer to do crosschain swaps would look as follows:
         "osmosis_swap": {
             "input_coin": {"denom":"token0","amount":"1000"}, 
             "output_denom":"token1",
-            "slippage":{"max_slippage_percentage":"5"},
+            "slippage":{"twap": {"slippage_percentage":"20", "window_seconds": 10}},
             "receiver":"juno1receiver",
-            "failed_delivery":null
+            "on_failed_delivery": "do_nothing",
```

Comment:
Presumably we also accept the key being missing, right?

Reply:
Not anymore. I can make it an option, but thought it was better to be explicit. Happy to change it either way

Reply:
Ah ok, that makes sense. Either is fine w/ me as well